src/pkg/lint/unsuccessful_validation/zarf.yaml

```diff
@@ -0,0 +1,21 @@
+kind: ZarfInitConfig
+metadata:
+  name: init
+  description: Testing bad yaml
+
+components:
+  - name: first-test-component
+    import:
+      not-path: packages/distros/k3s
+
+  - name: second-test-component
+    not-import:
+      path: packages/distros/k3s
+
+  - name: import-test
+    import:
+      path: "###ZARF_PKG_TMPL_ZEBRA###"
+
+  - name: import-url
+    import:
+      url: "oci://###ZARF_PKG_TMPL_ZEBRA###"
```

Comment:
We don't usually place test files here in `pkg`.  If we _need_ to we can reuse the `11-lint` test package(s) but should also look for a way to mock this out better if we can.

Reply:
I know just adding the files in plain text as a variable in go didn't work because our .editerconfig file automatically turns 4 spaces into a tab. 

I will see if any of our existing yaml packages allows for key values additions and see how we like adding schema errors that way

Reply:
Didn't try adding key values, could still do it but I feel it will be kinda ugly. Ended up using the zarf.yaml in the lint package for now, but agree that isn't ideal as someone might change that file and not notice the other test break. My vote would be to put it in plaintext, but then we have to change the .editerconfig to not turn spaces into tabs

Reply:
I ended up adding the file as plaintext in the test file.